test/geometry/adjoints.jl

```diff
@@ -63,3 +64,28 @@ end
         end
     end
 end
+
+@testset "DiffImages.Homography method gradient" begin
+    h = DiffImages.Homography{Float64}(SMatrix{3,3,Float64,9}([1 0 0;0 1 0;0 0 1]))
+    v = rand(SVector{2, Float64})
+    zy = Zygote.gradient((x,y)->x |> y |> sum, v, h)
```

Comment:
Isn't `(x,y) -> sum(y(x))` more readable here?

Reply:
Ahh yes, this completely skipped my eyes! I had removed `|>` from all the rest of the tests, should have removed it from here too. I'll change it right away.